EIPS/eip-3238.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 3238
+title: Vatnajoekull Difficulty Bomb Delay
+author: Afri Schoedon (@q9f)
+discussions-to: https://github.com/ethereum/EIPs/pull/3238
+type: Standards Track
+category: Core
+status: Draft
+created: 2021-01-25
+---
+
+## Simple Summary
+The average block times are increasing due to the difficulty bomb (also known as the "_ice age_") and slowly accelerating. This EIP proposes to delay the difficulty bomb for approximately ten months with the Vatnajoekull fork, a yet to be scheduled adjustment.
+
+## Abstract
+Starting with `VTNJKLL_FORK_BLKNUM` the client will calculate the difficulty based on a fake block number suggesting to the client that the difficulty bomb is adjusting eleven million blocks later than previously specified with the Homestead fork; that is two million blocks later than specified with the latest delay as included in the Muir Glacier fork ([EIP-2387](./eip-2387.md)).
+
+## Motivation
+Even after the Ethereum 2.0 mainnet launch, Ethash proof-of-work mining on the legacy chain should be feasible. It should allow miners sealing new blocks every 13~15 seconds on average for another ten months and allow both Ethereum 1.x and Ethereum 2.0 developers to conclude the merge.
+
+## Specification
+#### Relax Difficulty with Fake Block Number
+For the purposes of `calc_difficulty`, simply replace the use of `block.number`, as used in the exponential ice age component, with the formula:
+
+    fake_block_number = max(0, block.number - 11_000_000) if block.number >= VTNJKLL_FORK_BLKNUM else block.number
+
+## Rationale
+This will delay the ice age by another ~26 million seconds (approximately ~9.89 months), so the chain would be back at ~30 second block times in summer 2022. Hopefully, by then the Eth1-to-Eth2 merge will be concluded and the ice age fullfilled its task.
+
+## Backwards Compatibility
+This EIP is not forward compatible and introduces backwards incompatibilities in the difficulty calculation. Therefore, it should be included in a scheduled hardfork at a certain block number. It's suggested to consider this EIP either with or shortly after the Berlin hard-fork but not later than July 2021.
+
+Alternatively, in order to maintain stability of the system, a it can be considered to activate this EIP along with EIP-1559 fee market changes in a bundle. With the delay of the ice age, there is a desire to no further increase inflation and rather incentivize users to participate in proof-of-stake consensus instead. 
+
+## Test Cases
+The test cases in their logic do not differ significantly from [EIP-649](./eip-649.md), [EIP-1234](./eip-1234.md), or [EIP-2384](./eip-2384.md) and can be duplicated.
+
```

Comment:
Consider either removing this section entirely, or specifying some tests in this document.  This is a weak suggestion, and if you think there is significant value to developers to keep this mention I think that is fine, but I'm of the weak opinion that it doesn't add much value and shorter EIPs tend to be preferable over longer ones.

```suggestion
```